readthedocsext/theme/templates/mfa/authenticate.html

```diff
@@ -22,12 +22,22 @@
     {% csrf_token %}
     {{ form|crispy }}
     {{ redirect_field }}
-
     <div class="ui stackable relaxed text menu">
-      <a class="item" href="{% url "account_login" %}">{% trans "Cancel" %}</a>
+      <button class="ui button" type="submit" form="logout-from-stage">
+        {% trans "Cancel" %}
+      </button>
       <div class="right menu">
         <button class="ui primary button" type="submit">{% trans "Verify" %}</button>
       </div>
     </div>
   </form>
+
+  <!-- We can't have nested forms, so we need to have a separate form for "Cancel" -->
```

Comment:
Avoid using HTML comments unless we really need the output in the final HTML. Template comments should be used instead.

```suggestion
  {# We can't have nested forms, so we need to have a separate form for "Cancel" #}                                                                    
```